D:\a\1\s\src\System.Management.Automation\engine\remoting\client\remotingprotocolimplementation.cs(752,24): error CS0628: 'ClientRemoteSessionDSHandlerImpl.Dispose(bool)': new protected member declared in sealed type [D:\a\1\s\src\System.Management.Automation\System.Management.Automation.csproj]

--- a/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs
+++ b/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs
@@ -736,26 +736,14 @@ namespace System.Management.Automation.Remoting
         #region IDisposable
 
         /// <summary>
-        /// Public method for dispose.
+        /// Release all resources.
         /// </summary>
         public void Dispose()
         {
-            Dispose(true);
+            _transportManager.Dispose();
 
             GC.SuppressFinalize(this);
-        }
-
-        /// <summary>
-        /// Release all resources.
-        /// </summary>
-        /// <param name="disposing">If true, release all managed resources.</param>
-        protected void Dispose(bool disposing)
-        {
-            if (disposing)
-            {
-                _transportManager.Dispose();
-            }
-        }
+        }   
 
         #endregion IDisposable
 

